Jessie Smith/assignments/landing_page/templates/dojos.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Dojos</title>
+	<link rel="stylesheet" type="text/css" href="{{url_for('static', filename ='css/style.css')}}">
+</head>
+<body>
+	<form action='' method='POST'>
+		  First name:<br>
```

Comment:
<br>'s are forbidden. For real. Use div's or use labels. The last thing you want to have happen on a Python belt exam is to lose pnts for css/html mishaps! Don't let any of your webfun knowledge slip through the cracks ;-)